SimplyLinkedList.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+from Link import Link
```

Comment:
Could you explain this a bit further? Since I come from a Java background, I am used to having separate files for each class/object.